README.md

```diff
@@ -4,7 +4,7 @@
 
 # Create-UI
 
-Create-UI is the fastest way to generate a functional, [Vectara](https://vectara.com/)-powered sample codebase for a range of user interfaces:
+Create-UI is the fastest way to generate a [Vectara](https://vectara.com/)-powered sample codebase for a range of user interfaces:
 
 - **Semantic search.** A typical search UI.
```

Comment:
nit:

These feel better to me using colons instead of periods, i.e.

**Semantic search:** A typical search UI.
**Summarized semantic search:** A typical search UI preceded by a summary of the most relevant results. Perfect for users who want to scan an overview before digging deeper into interesting search results.

etc.